TypeCobol/Compiler/Scanner/MultilineScanState.SpecialNamesContext.cs

```diff
@@ -156,7 +157,7 @@ private void CreateCurrencyDescriptor()
                 }
 
                 //Helper local functions
-                string Text(Token alphanumericLiteralToken) => ((AlphanumericLiteralTokenValue) alphanumericLiteralToken.LiteralValue).Text;
+                string Text(Token alphanumericLiteralToken) => ((AlphanumericLiteralTokenValue) alphanumericLiteralToken.LiteralValue!).Text; //Using null-forgiving operator here as AlphanumericLiteralToken always has a non-null LiteralValue.
```

Comment:
How do you know that `LiteralValue` is never null here ?

Reply:
The token received here is guaranteed to be non-null of type `AlphanumericLiteral`, `HexadecimalAlphanumericLiteral`, `NullTerminatedAlphanumericLiteral`. All three are creatd by the `AbstractScanner.ScanAlphanumericLiteral` method which always sets the `LiteralValue` property.

Reply:
done